README.md

```diff
@@ -792,7 +793,8 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [mpd](http://www.musicpd.org/) - Daemon to remotely play music, stream music, handle and organize playlists. Many clients available. `GPLv2` `C++`
   * [mStream](http://mstream.io/) - Music streaming server with GUI management tools. Runs on Mac, Windows, and Linux. ([Demo](https://darncoyotes.mstream.io/), [Source Code](https://github.com/IrosTheBeggar/mStream)) `GPLv2` `Nodejs`
   * [Music Browser](https://github.com/henrik242/musicbrowser) - Browser and streamer for your music collection. It is runs on most operating systems, and is light enough to run flawlessly on NAS devices. `GPLv3` `PHP`
-   * [ympd](http://www.ympd.org/) - Standalone MPD Web GUI written in C, utilizing Websockets and Bootstrap/JS. ([Source Code](https://github.com/notandy/ympd)) `GPLv2` `C`
+  * [musikcube](https://musikcube.com/) - Terminal-based music player, library, and streaming audio server. ([Source code](https://github.com/clangen/musikcube)) `BSD` `C++`
```

Comment:
Terminal application, therefore not a web app/network service and does not qualify for the list.

Reply:
MusikCube *is* a network service, though. That's its entire point, to be a self-hosted network-capable music library manager that provides a music streaming server.

Reply:
I've read their documentation a little deeper, alright, can you change the description to

`Streaming audio server with Linux/OSX/Windows/Android clients.`

?